cluster/examples/kubernetes/ceph/csi/template/cephfs/csi-cephfsplugin.yaml

```diff
@@ -23,7 +23,7 @@ spec:
           args:
             - "--v=5"
             - "--csi-address=/csi/csi.sock"
-            - "--kubelet-registration-path=/var/lib/kubelet/plugins/cephfs.csi.ceph.com/csi.sock"
+            - "--kubelet-registration-path=/var/lib/kubelet/plugins/{{ .DriverNamePrefix }}cephfs.csi.ceph.com/csi.sock"
```

Comment:
I don't understand why @Madhu-1  is making this change. I didn't see any breakages in my PR due to the lack of this so I figured these were internal paths. If kube actually needs them to address CSI properly, that's fine but I'm curious why I didn't see issues during my testing.

Reply:
I wonder if the same instance of the csi driver can be used for attaching/mounting and only the provisioner needs to have the separate name?

Reply:
This is required if you are running multiple csi drivers, if multiple node plugins are listening on the same socket they request may go to any node plugin. to avoid this we need to have a unique socket per csi driver. it is recommended to have socket inside the driver name folder.

I have confirmed this issue   with below steps
note: I have only changed driver name as currently, we do in rook today, not the directory name
* deployed 2 rbd csi driver with different names rbd.csi.com and test.rbd.csi.com
* created PVC with rbd.csi.com
* created pod with PVC
* am able to see the device attachment logs in the second driver node plugin i.e test.rbd.csi.com